docs/cli/commands.md

```diff
@@ -521,6 +521,66 @@ tableland list
 ]
 ```
 
+## transfer
+
+Transfer a table to another address
+```markdown
+Positionals:
+  name      The target table name                            [string] [required]
+  receiver  The address to transfer the table to             [string] [required]
+
+Options:
+
+  -k, --privateKey           Private key string                         [string]
+  -c, --chain                The EVM chain to target                    [string]
+```
+
+### Example 
+
+Transfer a table from current wallet to another address
+
+```bash 
+tableland transfer example_table_313337_1 0x0000000000000000000000
+```
+
+Response:
+```JSON
```

Comment:
@dtbuchholz I put this response example in here because other commands had it. It doesn't seem especially useful, and it takes up a lot of space. Do you think we ought to keep it or toss it?

Reply:
@awmuncy I think it's fine to keep, for now. I'll probably rework that whole page a bit more, so I wouldn't worry about it.

Although, maybe that's indicative of implementing a different, condensed response? Just an idea but not needed to get this merged.